charts/spark-operator-chart/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: spark-operator
 description: A Helm chart for Spark on Kubernetes operator
 version: 1.0.9
```

Comment:
Please update the chart version.

Reply:
Updated to `1.0.10` or should it be `1.1.0` since spark 3.1.1?

Reply:
Yes, `1.1.0` sounds good given the Spark version change.